spec/models/reporting/workload_report_spec.rb

```diff
@@ -68,11 +68,7 @@
     end
   end
 
-  describe 'number of rows can be configured using :number_of_days' do
-    subject(:report) { described_class.new(number_of_rows: 7) }
-
-    it 'returns the counts for the four working days up to and including day_zero' do
-      expect(report.table.rows.size).to be 7
-    end
+  it 'number of rows defaults to 5' do
```

Comment:
Just thinking is this not a different test than a replacement? As in, it can be configured is one thing, and the fact it is defaulted to 5 is another.

Could we have both tests? or do you think that is overkill?

Reply:
I've updated the other specs to use a configured rather than default value, so that case is handled there. This is just for the default.